boa/src/value/mod.rs

```diff
@@ -527,11 +527,27 @@ impl Value {
         // 1. Assert: input is an ECMAScript language value. (always a value not need to check)
         // 2. If Type(input) is Object, then
         if let Value::Object(obj) = self {
+            let exotic_to_prim = obj.get(
```

Comment:
we should use the `GetMethod` function (`.get_method()`) as described in the [spec](https://tc39.es/ecma262/#sec-toprimitive)

Reply:
Done!